src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/IPackageMetadata.cs

```diff
@@ -29,6 +29,7 @@ public interface IPackageMetadata
         bool Serviceable { get; }
         string Copyright { get; }
         string Icon { get; }
+        string Readme { get; }
```

Comment:
A concern I have is that one could easily mistake `Readme` as the readme content instead of a path. What do you think of renaming properties to `ReadmePath` and methods to `GetReadmePath()`?

Reply:
Would this change apply to the actual nuspec element name `<readme>` as well, making it something like `<readmePath>`, or would it only apply to the names of properties and methods in the code?

Reply:
Up to you. Personally I'm fine with just the properties and methods in code so that we can keep the nuspec element `<readme>` consistent with the `<license type="file">` element.

Reply:
I agree on not changing the nuspec element name. However, I'm personally unsure about how we should weigh the concerns about the `Readme` property name not being clear on readme contents vs path, against possible concerns about consistency between:

1. the nuspec `<readme>` element and other nuspec elements (we seem to agree that we shouldn't change this)
2. the nuspec `<readme>` element and the corresponding `Readme` properties in the code

Would `ReadmePath` properties in the code cause confusion because of the fact that the corresponding nuspec element is named `<readme>` and not `<readmePath>`? I don't necessarily have an answer to this, and I'm happy to leave this conversation open for others to weigh in too.